sdk/python/tests/unit/test_feature_service.py

```diff
@@ -5,6 +5,7 @@
 from feast.field import Field
 from feast.infra.offline_stores.file_source import FileSource
 from feast.types import Float32
+from tests.utils.test_wrapper_utils import no_warnings
```

Comment:
too many `utils`

Reply:
I renamed it since it doesn't really have to be a utils but I still think having a file with test_wrappers allows code reusability